src/main/java/com/powsybl/openloadflow/dc/DcTargetVector.java

```diff
@@ -24,7 +25,12 @@ public class DcTargetVector extends TargetVector<DcVariableType, DcEquationType>
     public static void init(Equation<DcVariableType, DcEquationType> equation, LfNetwork network, double[] targets) {
         switch (equation.getType()) {
             case BUS_TARGET_P:
-                targets[equation.getColumn()] = network.getBus(equation.getElementNum()).getTargetP();
+                LfBus bus = network.getBus(equation.getElementNum());
+                targets[equation.getColumn()] = bus.getTargetP();
+                // Only used for multi slack (BUS_TARGET_P equation is disabled for first slack bus)
+                if (bus.isSlack()) {
```

Comment:
Hi @jeandemanged and @vmouradian, reading this line with @geofjamg today, I'am not sure your implementation is resilient to disabled buses (slack or not). I think a test case with a connectivity loss (including a participating element) could be imagine to verify this. Think about the case where we loose a slack bus, I don't remember what we support...